bot_control/PathArray[] agent